Toast/toast.css

```diff
@@ -0,0 +1,65 @@
+@import url(/styles/reset.css);
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600&display=swap');
+
+:root {
+    --text-color-primary: #e4e4e7;
+    --text-color-secondary: #6366f1;
+    --background-color: #27272a;
+    --hover-color: #52525b;
+}
+
+body {
+    font-family: 'Raleway', sans-serif;
+}
+
+.baseline {
+    bottom: 3rem;
+    left: 40%;
+}
+
+.leading {
+    bottom: 3rem;
+    right: 1rem;
+}
+
+.flex {
+    display: flex;
+    gap: 1.3rem;
+    align-items: center;
+}
+
+.fixed {
+    position: fixed;
+}
+
+.toast {
+    background-color: var(--background-color);
+    color: var(--text-color-primary);
+    padding: 0.35rem 1rem;
+    font-size: 1rem;
+    border-radius: 4px;
+}
+
+.close {
+    border-radius: 50%;
+    height: 36px;
+    width: 36px;
```

Comment:
You can use rem, any specific reason to use px?

Reply:
thanks, updated.